src/main/java/com/github/havardh/javaflow/phases/reader/FileReader.java

```diff
@@ -23,10 +25,8 @@
     try {
       return of(new String(Files.readAllBytes(Paths.get(filename))));
     } catch (IOException e) {
-      e.printStackTrace();
-      System.exit(1);
+      throw new ExitException(ErrorCode.COULD_NOT_PARSE_SOURCE_CODE, e);
```

Comment:
The `FileReader` does not parse, so I propose to make this `ErrorCode.COULD_NOT_READ_FILE`.

Reply:
Good point, fixed!